cnxarchive/utils/ident_hash.py

```diff
@@ -40,10 +40,10 @@ def split_ident_hash(ident_hash, split_version=False):
     if HASH_CHAR not in ident_hash:
         ident_hash = '{}@'.format(ident_hash)
     split_value = ident_hash.split(HASH_CHAR)
-    if split_value[0] == '':
-        raise ValueError("Missing values")
 
     try:
+        if split_value[0] == '':
+            raise ValueError("Missing values")
```

Comment:
I'm sure this works, but I strongly feel that it is incorrect. you shouldn't catch it and raise it as an ``IdentHashSyntaxError``, because the value is missing, which means there is no syntax to error on. Therefore, it's legitimately an error with the value (aka ValueError), not the syntax.